services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/api/_resource_tracker.py

```diff
@@ -0,0 +1,108 @@
+import logging
+from typing import Annotated, Literal
+
+from fastapi import APIRouter, Depends, status
+from fastapi_pagination.api import create_page
+from models_library.api_schemas_resource_usage_tracker.credit_transactions import (
+    WalletTotalCredits,
+)
+from models_library.api_schemas_webserver.resource_usage import (
+    PricingPlanGet,
+    ServiceRunGet,
+)
+from models_library.resource_tracker import CreditTransactionId
+
+from ..models.pagination import LimitOffsetPage, LimitOffsetParamsWithDefault
+from ..models.resource_tracker_service_run import ServiceRunPage
+from ..services import (
+    resource_tracker_credit_transactions,
+    resource_tracker_pricing_plans,
+    resource_tracker_service_runs,
+)
+
+logger = logging.getLogger(__name__)
+
+
+router = APIRouter()
+
+
+########
+# USAGE
+########
+
+
+@router.get(
+    "/usage/services",
```

Comment:
here the resource you give back is the usage, not the services.
What about ```/services/-/usages``` ?? I think that would go wit the usual APIs we create

Reply:
thanks, good point you are absolutely right! I will change it on resource-usage-tracker side. In the webserver, frontend is already integrated so i am not sure whether I should change it. I will see how much work it would be.